src/graph/graph-state-manager.ts

```diff
@@ -101,81 +86,99 @@ export class GraphStateManager implements OnApplicationBootstrap {
     return Graph.deserializeDsnpKeys(keys);
   }
 
-  public importUserData(payload: ImportBundle[]): boolean {
-    if (this.graphState) {
-      return this.graphState.importUserData(payload);
+  // eslint-disable-next-line class-methods-use-this
+  public importUserData(graphState: Graph, payload: ImportBundle[]): boolean {
+    if (graphState) {
+      return graphState.importUserData(payload);
     }
     return false;
   }
```

Comment:
Can probably just get rid of this method; client can call `graphState.importUserData(payload)` directly.

Reply:
In general, continuing below, I think any method just wrapping a call to the `Graph` object & not using any of this class's internal attributes, can just be eliminated.